assets/airports/cyhz.json

```diff
@@ -0,0 +1,657 @@
+{
+    "radio": {
+        "twr": "Halifax Tower",
+        "app": "Halifax Arrival",
+        "dep": "Halifax Departure"
+    },
+    "icao": "CYHZ",
+    "iata": "YHZ",
+    "magnetic_north": -18,
+    "ctr_radius": 80,
+    "ctr_ceiling": 12000,
+    "initial_alt": 5000,
+    "position": ["N44d52m52.73", "W063d30m33.62", "477.0ft"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N44d52m47.00", "W063d30m37.00"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 37,
+        "speed": 9
+    },
+    "arrivalRunway": "05",
+    "departureRunway": "05",
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 12000,
+            "airspace_class": "E",
+            "poly": [
```

Comment:
This airspace shape seems a little fishy... does NavCanada have any AIP type thing, or are all their charts paid? I think it actually may be the latter, they're pretty stingy (read "cheap").

In my world, the pink shading actually indicates area where the floor of class E (lowest controllable class of airspace) is extended lower than normal (I think to 700AGL instead of 1200AGL... something like that). So that's probably what that is, and the Class B/C airspace would be the blue lines. Of course, that's only the airspace of an elevated class, while the approach control facility (if there is one after all) probably owns additional non-elevated airspace (Class E), but as always that part can be difficult or impossible to find. ☹️